lock-api/src/main/java/com/palantir/lock/client/LockRefreshingLockService.java

```diff
@@ -38,8 +38,7 @@
 import com.palantir.lock.SimpleHeldLocksToken;
 import com.palantir.lock.SimplifyingLockService;
 
-@SuppressWarnings("checkstyle:FinalClass") // Avoid breaking API in case someone extended this
```

Comment:
just so we're clear I did look and the only project who could have done this, didn't.
It won't cause a break and make @mswintermeyer fret